python-avd/pyavd/_eos_cli_config_gen/schema/schema_fragments/snmp_server.schema.yml

```
@@ -142,14 +144,17 @@ keys:
             notify:
               type: str
               description: Notify view.
+      # TODO: AVD 6.0, consider separating local and remote users in model.
       users:
         type: list
         items:
           type: dict
           keys:
+            # TODO: AVD 6.0, make this primary_key.
```

```suggestion
            # TODO: AVD 6.0, make this required.
            # Cannot make it primary key because same user group can be defined with different version
```

would need composite key ..

```
pf1(config-s-blaaa)#show session-config diff
--- system:/running-config
+++ session:/blaaa-session-config
+snmp-server engineID local f5717f500000d8ac1900
+snmp-server user ba be v1
+snmp-server user ba be v2c
+snmp-server user ba be v3
```